ray-operator/config/samples/ray-service.autoscaler.yaml

```
@@ -17,10 +17,10 @@ spec:
   serveConfigV2: |
     applications:
       - name: app1
-        import_path: autoscaling_test.blocked.app
+        import_path: autoscaling.blocked.app
```

https://github.com/ray-project/kuberay/pull/1175#discussion_r1236013964

```suggestion
        import_path: autoscaling.blocked:app
```